try:
    a= int(input("Enter your no ."))
    print(a+3)
#except:
 #   print(" Some error occured")
except Exception as e:
    print(" Some error occured" ,e )
